.github/workflows/nodejs.yml

```diff
@@ -1,31 +1,49 @@
-name: 'tests'
-
-on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+on: [push, pull_request]
```

Comment:
I'm not sure.
Should we separate in another workflow the publish step? We don't want to publish for PRs, only for commits on master. :smile:

Reply:
Probably. :laughing: I'll separate testing and publishing into 2 workflows.